src/components/UserProfile/TeamsAndProjects/AddTeamPopup.jsx

```diff
@@ -41,14 +73,20 @@ const AddTeamPopup = React.memo(props => {
             Confirm
           </Button>
         </div>
-        <div>
-          {!isValidTeam && selectedTeam && (
```

Comment:
Why was this block removed? (45-47) I feel like this message is necessary if the user chooses a team that the person is already assigned to.